samples/unit-testing/sample.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Unit Testing NServiceBus 6
+summary: How to write unit tests for NServiceBus 6 systems
+tags:
+- Unit Testing
+- Arrange-Act-Assert
+- NUnit
+redirects:
+related:
+- nservicebus/testing
+---
+
+This sample shows how to write unit tests for various NServiceBus components using "bare-metal", Arrange-Act-Assert (AAA) style tests. This sample is a test project that uses NUnit, and utilizes testable helper implementations from the `NServiceBus.Testing` package.
```

Comment:
not sure if we really wanna use the word "bare metal" tests on the documentation.